pom.xml

```diff
@@ -108,6 +108,13 @@
       <version>2.28.0</version>
       <scope>test</scope>
     </dependency>
+  
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.9</version>
+    </dependency>
```

Comment:
I'm pushing a commit to get rid of this dependency. No need to add it for a single method.